ePrint IACR.js

```diff
@@ -56,8 +59,7 @@ function scrape(doc, url) {
 	
 	newItem.date = year;
 	newItem.reportNumber = no;
-	//we want to use this later & make sure we don't make http--> https requests or vice versa. 
-	newItem.url = url.match(/^https?:\/\/[^\/]+/)[0] + "/" + year + "/" + no;
+	newItem.url = "https://eprint.iacr.org/" + year + "/" + no;
```

Comment:
I still get regular http, e.g. at http://eprint.iacr.org/2005/033 -- please revert this.